packages/shared/src/components/widgets/SocialShareButton.tsx

```diff
@@ -22,42 +18,36 @@ const sizeToText = {
   [ButtonSize.Medium]: 'typo-caption1',
 };
 
-export const SocialShareIcon = ({
-  pressed,
+export const SocialShareButton = ({
   href,
   icon,
-  className,
-  onClick,
   label,
-  disabled,
   size = ButtonSize.Large,
-}: SocialShareIconProps): ReactElement => {
+  ...props
+}: SocialShareButtonProps): ReactElement => {
   const button = useRef<HTMLButtonElement>();
-  const buttonProps = href
-    ? ({
-        href,
-        rel: 'noopener',
-        target: 'blank',
-        tag: 'a',
-        disabled,
-      } as ButtonProps<'a'>)
-    : ({ onClick, disabled } as ButtonProps<'button'>);
+  const buttonProps =
+    href &&
+    ({
+      href,
+      rel: 'noopener',
+      target: 'blank',
+      tag: 'a',
+    } as ButtonProps<'a'>);
 
   return (
     <div className="flex flex-col items-center w-16">
       <Button
         {...buttonProps}
+        {...props}
         data-testid={`social-share-${label}`}
-        buttonSize={size}
-        className={className}
-        iconOnly
+        size={size}
         icon={icon}
-        pressed={pressed}
         ref={button}
       />
       <ShareText
         className={classNames(
-          'mt-1.5 max-w-[4rem] overflow-hidden overflow-ellipsis text-center',
+          'mt-1.5 max-w-16 overflow-hidden overflow-ellipsis text-center', // max-w-[4rem]
```

Comment:
You have a comment here still :)
Also you verified the max-w-16 works? (not sure why we had the arbitraty one before then.

Reply:
Good catch. I added the 16 as an arbitrary value in [tailwind.config.js](https://github.com/dailydotdev/apps/pull/2448/files#diff-23b6dee9318cba31995fad41ac7e2579aa617078b5963bcb8d94e6d75cda6159R212)

Reply:
It seems like `max-w-16` class was not being precomputed for us by tailwind though. But it should. not matter, since the parent component has `w-16` class, so this child will never exceed that anyway. I think it should be safe to remove the `max-w-` altogether, so that's what I'll do here.